javaslang/src/main/java/javaslang/control/GwtIncompatible.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2013 Google Inc.
```

Comment:
> ?

? :smile: 
This class is part of GWT, just copied to Javaslang to avoid additional dependency